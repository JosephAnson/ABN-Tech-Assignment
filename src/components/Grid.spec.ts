import { render } from "@testing-library/vue"
import { describe, expect, test } from "vitest"
import Grid from "./Grid.vue"

describe("Grid", () => {
  test("should render grid", () => {
    const { html } = render(Grid)
    expect(html()).toMatchSnapshot()
  })
})
