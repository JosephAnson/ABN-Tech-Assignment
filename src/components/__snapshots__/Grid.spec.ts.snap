// Vitest Snapshot v1

exports[`Grid > should render grid 1`] = `"<div class=\\"grid grid-cols-2 gap-6 sm:grid-cols-4 sm:gap-6 lg:grid-cols-6 lg:gap-6\\"></div>"`;
